src/material-examples/radio-overview/radio-overview-example.css

```diff
@@ -1,3 +1,3 @@
 .mat-radio-button ~ .mat-radio-button {
-  padding-right: 16px;
+  padding-left: 16px;
```

Comment:
Can you try switching this to margin? We had a similar PR https://github.com/angular/material2/pull/13470